pkg/assert/pod.go

```diff
@@ -57,25 +56,7 @@ func checkReadyFields() types.GomegaMatcher {
 	}, BeTrue())
 }
 
-// PodAssertStatus custom assertion that asserts that pod status is completed or in some cases
-// apply pods can have an error status
-func PodAssertStatus() PodAssertFunc {
```

Comment:
this is removed due to we are already doing this validation on 

https://github.com/rancher/distros-test-framework/blob/8329d5ac4184f0378e973304a0f8164988cb1bd0/pkg/testcase/pod.go#L41